src/libcaf-gfortran-descriptor.h

```diff
@@ -51,6 +51,9 @@ typedef struct gfc_descriptor_t {
   void *base_addr;
   size_t offset;
   ptrdiff_t dtype;
+#if (__GNUC__ >= 8)
```

Comment:
@vehre Is there anything that would be useful for a Fortran programmer to know about the behavior of OpenCoarrays with  GCC >= 8?  If so, let's document the differences somewhere.  If it's in a commit message, then it's likely to make it into the next release notes.  If not, then I'm not sure of the best place but probably either README.md or INSTALL.md.